src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -890,6 +891,42 @@ class C { }
             AssertEx.Equal(results.Diagnostics, fullDiagnostics);
         }
 
+        [Fact]
+        public void Cancellation_During_Initialization_Doesnt_Report_As_Generator_Error()
```

Comment:
```suggestion
        public void Cancellation_During_Initialization_Does_Not_Report_As_Generator_Error()
```

Reply:
Leaving this one since it follows the naming pattern established by unaltered test `Cancellation_During_Execution_Doesnt_Report_As_Generator_Error`